1.18.1

```diff
@@ -0,0 +1 @@
+Requirement already satisfied: numpy in d:\apps\anaconda3\lib\site-packages (1.16.2)
```

Comment:
This file should not be committed with the PR.

Reply:
Sorry about that, I was still getting used to the whole thing.

Reply:
Is there a way to remove them from the PR ? Or are will you do it yourself when merging ?

Reply:
I just deleted them and commited the change from the github interface!